usaspending_api/common/etl/operations.py

```diff
@@ -0,0 +1,179 @@
+""" Functions to generate SQL for several higher level ETL operations. """
+
+
+from psycopg2.sql import Literal, SQL
+from typing import List
+from usaspending_api.common.etl import ETLTable, primatives
+from usaspending_api.common.helpers import sql_helpers
+
+
+def _get_shared_columns(columns: List[str], other_columns: List[str]) -> List[str]:
+    """ Return the list of columns contained in both lists. """
+
+    shared_columns = [c for c in columns if c in other_columns]
```

Comment:
My mind naturally thinks of `set()` for these types of operations. Nothing wrong with list comp here

Reply:
Mine too.  I first wrote it that way, but I think this might be more efficient than casting back and forth between sets.

Reply:
For small lists the costs probably won't be noticeable.

Sets would change the behavior if the columns list contained duplicates. Unsure if duplicates are desired/expected/possible